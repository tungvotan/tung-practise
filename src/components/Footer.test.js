import React from 'react';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

const setup = (setupProps = {}) => {
  const wrapper = shallow(<Footer/>);

  return {
    wrapper
  };
};

describe('Footer component', () => {
  test('renders without crashing', () => {
    const { wrapper } = setup();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
