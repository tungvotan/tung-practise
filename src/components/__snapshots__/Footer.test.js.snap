// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders without crashing 1`] = `
<div>
  <span>
     Show: 
  </span>
  <Connect(Link)
    filter="SHOW_ALL"
  >
     All 
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_ACTIVE"
  >
     Active 
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_COMPLETED"
  >
     Completed 
  </Connect(Link)>
</div>
`;
